src/main/asyncapi/network_survey_messaging.yaml

```diff
@@ -221,6 +224,32 @@ components:
       payload:
         $ref: '#/components/schemas/lteNasPayload'
 
+    umtsNas:
+      name: UmtsNas
+      title: UMTS NAS Message
+      summary: Represents a raw UMTS NAS message sent Over The Air (OTA) between an UMTS uNodeB and an UMTS UE.
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/umtsNasPayload'
+
+    umtsNasDsds:
```

Comment:
I don't think an entire new message is needed for UMTS NAS DSDS. The parsing is different for the message because of the SIM ID at the beginning, but given we are going to include the full pcap record I don't think we need a new message just for DSDS. Even if we are not going with the full pcap record, I think we could get away with just using a different messageType field value (e.g. UmtsNas vs UmtsNasDsds).